build-lib/lib.js

```diff
@@ -95,12 +89,13 @@ window.ShopifyBuy = require('shopify-buy/shopify').default;
     const minifiedTree = uglifyJavaScript(funnel(tree, {
       getDestinationPath: function (path) {
         return path.replace(/\.js/, '.min.js');
-      }
+      },
+      exclude: ['**/*.map']
```

Comment:
Nope. The `concat` module used earlier already generates source maps. This is to tell `uglify` not to generate its own new set of source maps. If we leave it so, a duplicate file error would be thrown.